docs/source/quickstart.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8a7e5a3c",
+   "metadata": {},
+   "source": [
+    "# x. Quickstart"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f25c2716",
+   "metadata": {},
+   "source": [
+    "RADICAL-Pilot (RP) is a Python module which can be installed via one of the following package managers **pip**, **conda**, or **spack**. RP is one of the building blocks of the RADICAL-Cybertools (RCT). RP **must** be installed in a virtual environment and **cannot** be installed as a system-wide Python package. \n",
+    "\n",
+    "It is required to have a **Python** installation of version **3.6+** (tested with **3.6**, **3.7**, **3.8**, **3.9**), RP dependecies will be installed either automatically during the installation process or could be pre-installed beforehand by the user."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "913159d8",
+   "metadata": {},
+   "source": [
+    "# x.1. Setup per different virtual environment flavor"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f84cf1f",
+   "metadata": {},
+   "source": [
+    "### x.1.1. `venv`"
+   ]
+  },
+  {
+   "cell_type": "raw",
```

Comment:
"raw" cell type is not shown in GitHub viewer... (if is opened within GitHub these cells will look empty)